src-db/database/sourcedata/AD_COLUMN.xml

```diff
@@ -837,8 +837,8 @@
 <!--53271001353E44A0A04168C9276E2FFC-->  <AD_CLIENT_ID><![CDATA[0]]></AD_CLIENT_ID>
 <!--53271001353E44A0A04168C9276E2FFC-->  <AD_ORG_ID><![CDATA[0]]></AD_ORG_ID>
 <!--53271001353E44A0A04168C9276E2FFC-->  <ISACTIVE><![CDATA[Y]]></ISACTIVE>
-<!--53271001353E44A0A04168C9276E2FFC-->  <NAME><![CDATA[Is External Dependency]]></NAME>
-<!--53271001353E44A0A04168C9276E2FFC-->  <COLUMNNAME><![CDATA[External_Dependency]]></COLUMNNAME>
+<!--53271001353E44A0A04168C9276E2FFC-->  <NAME><![CDATA[External Dependency]]></NAME>
+<!--53271001353E44A0A04168C9276E2FFC-->  <COLUMNNAME><![CDATA[isexternal_dependency]]></COLUMNNAME>
```

Comment:
Maybe it should be `isexternaldependency`? I put it like that in my issue branch :grimacing: